docs/java/how-to-develop-a-worker-program-in-java.md

```diff
@@ -18,7 +18,9 @@ Then register all Workflow Types and all Activity Types that the Worker can exec
 For Activities, since they are stateless and thread-safe, instances are used in the registration process instead of the Java Class.
 A Worker can be registered with just Workflows, just Activities, or both.
 
-This single Worker Process can contain many Workers Objects.
+## Create Worker
```

Comment:
Let's drop all of these to h4s so that they work inside the app-dev-guide

Reply:
done